content/docs/sidebar.json

```diff
@@ -730,14 +730,8 @@
               "remove-a-model-or-its-details"
             ]
           },
-          {
-            "slug": "team-collaboration",
-            "children": ["change-team-plan-and-size"]
```

Comment:
[F] `change-team-plan-and-size` has been deleted. There is now a `Get Enterprise` section at the bottom of `team-collaboration`. All existing links should point to that section.